package.json

```diff
@@ -109,9 +111,13 @@
   "peerDependencies": {
     "react": "^18.2.0",
     "react-dom": "^18.2.0",
+    "react-icons": "^5.4.0",
     "tailwindcss": "^3.4.1"
   },
   "release": {
     "extends": "./release.config.js"
+  },
+  "dependencies": {
+    "tailwind-merge": "2.6.0"
```

Comment:
You've made this a dependency, but `react-icons` a peer dependency, I'd expect these to be the same.